app/components/MixerVolmeter.vue.ts

```diff
@@ -266,7 +292,15 @@ export default class MixerVolmeter extends TsxComponent<MixerVolmeterProps> {
 
     this.gl.uniform2f(this.scaleLocation, 1, CHANNEL_HEIGHT);
     this.gl.uniform2f(this.translationLocation, 0, channel * (CHANNEL_HEIGHT + PADDING_HEIGHT));
-    this.gl.uniform1f(this.volumeLocation, this.dbToUnitScalar(peak));
+
+    const prevPeak = this.prevPeaks ? this.prevPeaks[channel] || peak : peak;
+    const timeDelta = performance.now() - this.lastEventTime;
```

Comment:
this looks like the first time `lastEventTime` is referenced but it's uninitialized, so wouldn't this be `performance.now() - undefined` in the first cycle?